custom_components/unifi_insights/sensor.py

```
@@ -533,8 +537,126 @@ def _calculate_storage_available(nvr_data: dict[str, Any]) -> float | None:
+    ),
+)
+
+
+def _outlet_has_metering(outlet: dict[str, Any]) -> bool:
+    """Return True when an outlet has metering capability or telemetry."""
+    caps = outlet.get("outlet_caps") or outlet.get("outletCaps")
+    if caps is not None:
+        try:
+            if int(caps) & 2:
+                return True
+        except TypeError, ValueError:
```

Pylint (4.0.7)

[error] Parsing failed: 'multiple exception types must be parenthesized (unifi_insights.sensor, line 639)'

(E0001)